main.go

```diff
@@ -307,10 +315,35 @@ func (m model) Update(msg tea.Msg) (tea.Model, tea.Cmd) {
 				m.textarea.Blur()
 				return m, tea.Quit
 			}
-
 		default:
-			if !m.textarea.Focused() {
-				cmd = m.textarea.Focus()
+			if m.textarea.Focused() {
+				m.textarea, cmd = m.textarea.Update(msg)
+				cmds = append(cmds, cmd)
+			}
+		}
+
+		// Command mode
+		if !m.textarea.Focused() {
+			switch msg.String() {
+			case "i":
+				m.textarea.Focus()
+			case "y":
+				if len(lastResponse) == 0 {
+					break
+				}
+				err := clipboard.WriteAll(lastResponse)
+				if err != nil {
+					panic("could not write to clipboard")
+				}
+			case "p":
+				m.textarea.Focus()
+				clip, err := clipboard.ReadAll()
+				msg.Runes = []rune(clip)
+				if err != nil {
+					panic("could not write to clipboard")
```

Comment:
Could not read from clipboard*

Reply:
Can you check if pasting works on your machine now? `golang-design/clipboard` says it only depends on X11, the previous library depended on xclip.